package.json

```diff
@@ -20,7 +20,7 @@
     "lint": "eslint --fix src",
     "prepublishOnly": "yarn build",
     "test": "mocha",
-    "test:e2e": "yarn run test:e2e:mainnet && yarn run test:e2e:polygon && yarn run test:e2e:arbitrum",
+    "test:e2e": "yarn run test:e2e:mainnet && yarn run test:e2e:arbitrum",
     "test:e2e:mainnet": "env-cmd -f .env.mainnet hardhat test --grep 'mainnet'",
     "test:e2e:polygon": "env-cmd -f .env.polygon hardhat test --grep 'polygon'",
```

Comment:
If we don't have any polygon tests, is it a good idea to remove this line and `.env.polygon`?

Reply:
Fixed it. My mistake, I removed it during testing and forgot to add it back.